cwltool/argparser.py

```diff
@@ -903,6 +903,11 @@ def add_argument(
             action = DirectoryAppendAction
         else:
             action = AppendAction
+            items = inptype["items"]
+            if items == "int" or items == "long":
+                atype = int
+            elif items == "double" or items == "fload":
```

Comment:
```suggestion
            elif items == "double" or items == "float":
```

needs a test with `type: float` to catch this typo :-)

Reply:
Thank you for correction. Fixed!